### PR TITLE
fix: accept 0 as a valid value

### PR DESCRIPTION
**Pull request by severo**
It allows to specify 

```
minimum: 0
```

### CODE REVIEW

**Comment by severo** [comment]
Hi, I'm sorry you thought this PR is hacktoberfest's spam, and I understand it's a bad month to contribute with small PR, but this commit is legit. Currently the user case I put in the description (`minimum: 0`) is ignored, while it's perfectly valid in JSONSchema.

My commit is because you want to test if the key is present in the object (I might have tested it with `if (key in schema)` instead), not if the value is falsy.

**Comment by saibotsivad** [comment]
Thanks for understanding. October is indeed a bad month. I'll drop the label and comment on the PR 🙇

File: index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ function generatePropertyRestrictions(schema) {
 
 function generateSinglePropertyRestriction(schema) {
 	return function(key, text) {
-		if (schema[key]) {
+		if (schema.key !== undefined) {
```
Comment (saibotsivad) [inline index.js]: You probably mean this instead?

```js
if (schema[key] !== undefined) {
```

Comment (severo) [reply index.js]: Sure!

**Pull request closed without merging**